### PR TITLE
Add GolangCI-Lint + Coveralls actions

### PR DESCRIPTION
**Pull request by ruudk**
* Add GolangCI linter
* Add Coveralls integration
* Replace status badge with Github Actions
* Run Github Actions tests also on commit (for example push to branch master)
* Removed Travis

### CODE REVIEW

**Comment by ruudk** [comment]
Github Actions is already enabled, great, so now I can try to get this green... :)

**Comment by ruudk** [comment]
@mikkeloscar @szuecs Ready for review :) 

Github Actions is so fast, love it.

I also removed Travis in this PR. 

Could you please fix the required branch so that Github Actions is required?

Image 2020-04-04 at 7 04 24 PM

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -10,6 +10,16 @@ jobs:
...
       with:
         go-version: '^1.14'
+    - run: echo "::add-path::$(go env GOPATH)/bin"
     - run: go version
+    - run: go get github.com/mattn/goveralls
+      env:
+        GO111MODULE: off
+    - run: curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin ${GOLANGCI_RELEASE}
+      env:
+        GOLANGCI_RELEASE: v1.21.0
```
Comment (mikkeloscar) [inline .github/workflows/ci.yaml:20]: Could we bump to the latest release now we're at it? https://github.com/golangci/golangci-lint/releases/tag/v1.24.0

**Comment by mikkeloscar** [comment]
@ruudk Currently we have some problems with the repo permissions so I can't change anything and have to go through our Org. I'll try to get this resolved today, but maybe it takes a bit before we can make all the changes on our side.

Thanks for the effort here.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (ruudk) [reply .github/workflows/ci.yaml:20]: Please use suggestions for this, then I can easily apply :)
```suggestion
        GOLANGCI_RELEASE: v1.24.0
```

**Comment by mikkeloscar** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**